packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -101,7 +101,13 @@ export interface InteriorSequencePlace {
  * @remarks See {@link SequencePlace} for additional context on usage.
  */
 export enum Side {
+	/**
+	 * Content inserted will be placed before the endpoint
```

Comment:
```suggestion
	 * Content inserted will be placed before the character
```

Perhaps "character"? Open to other words as well. I think endpoint doesn't fit here.

Reply:
The comments I added are not helpful. I'll remove them.